crates/core/tests/fixtures.rs

```diff
@@ -0,0 +1,98 @@
+use ethers::abi::Token;
+use ethers::types::Address as EthersAddress;
+use futures::executor::block_on;
+use kakarot_rpc_core::client::KakarotClient;
+use kakarot_rpc_core::models::felt::Felt252Wrapper;
+use kakarot_rpc_core::test_utils::deploy_helpers::{ContractDeploymentArgs, DeployedKakarot, KakarotTestEnvironment};
+use reth_primitives::Address;
+use rstest::*;
+use starknet::providers::jsonrpc::HttpTransport;
+use starknet::providers::JsonRpcClient;
+
+/// This is a the context for a Kakarot test environment
+/// It is a wrapper around the KakarotTestEnvironment
+pub struct KakarotTestEnvironmentContext {
```

Comment:
I'm wondering if instead of having this wrapper we should maybe just remove `KakarotTestEnvironment` and only keep `KakarotTestEnvironmentContext`, what do you think?

Reply:
I was thinking about this as well, I think it can be a good idea 💪